integration_tests/__init__.py

```diff
@@ -31,6 +33,7 @@
 from testtools.matchers import MatchesRegex
 
 from snapcraft.tests import fixture_setup
+from snapcraft.internal.repo import _deb
```

Comment:
this gives me a bad vibe, not sure how to solve it yet. Maybe reimplement here, @elopio ?